Automattic/jetpack
# My Jetpack: Add redux store for my-jetpack

Closes #22334

#### Changes proposed in this Pull Request:

* Adds a new store for my-jetpack matters.
* Introduces minimal method returning empty array for initial state of products. (Scaffolding for follow-up PR). 
* Introduces some scaffolding files like `actions.js`. Follow-up PR will use those files.

#### Does this pull request change what data or activity we track or use?
none
#### Testing instructions:

* Checkout this branch
* Build with `jetpack build packages/my-jetpack`.
* Have Jetpack Backup and a plan for it (either Upgrade to Security or Backup). 
* Make sure the constant `JETPACK_ENABLE_MY_JETPACK` is set.
* Checkout the My Jetpack page.
* Open the Redux Dev tools. Select the `my-jetpack` store, switch to the State tab and confirm you see data under the `root` key.
  
![image](https://user-images.githubusercontent.com/746152/149505269-f48e0a28-4b27-4114-a03c-b2ed4534772e.png)
